### ISSUE TITLE
feature: backup/sync to a name given with parameter to specify ip address

### ISSUE THREAD

**Issue by hokreb** [hersle/rmirro#13]
on my system the ip address of the remarkable web interface is not always the same. Maybe it would be nice to specify a directory for the backup/meta/rendered data and optionally add an ip address for the ssh access for it. By this changing ip addresses would have no impact

**Comment by hersle** [comment]
So maybe an option like `rmirro.py --name=myremarkable 123.456.7.89` that overrides the directory name from `123.456.7.89_backup` to `myremarkable_backup` etc.?

**Comment by Penguin-Guru** [comment]
Yes, this would be very nice. Rather than just the directory name, perhaps a full path to an arbitrary directory.

As for the changing I.P. address, that sounds like a case for using a host name.
